sdk/attestation/azure-security-attestation/samples/attestation/attest_openenclave_shared.cpp

```diff
@@ -47,7 +47,7 @@ int main()
     std::string const endpoint
         = "https://shared" + shortLocation + "." + shortLocation + ".attest.azure.net";
 
-    AttestationClient const attestationClient(AttestationClient::Create(endpoint));
```

Comment:
We have const in other places, I thik we should keep it here, if it compiles.

Reply:
For the samples, I went back to the original state of the various samples before I started tweaking the factory methods. This is the way the sample was when it was originally written.

And honestly, I expect customers will usually use the non-const form - that seems to be the norm throughout most of the Azure SDK.